web3.go

```diff
@@ -64,7 +64,9 @@ type Network struct {
 
 var (
 	weiPerGO   = big.NewInt(1e18)
+	OneWei     = weiPerGO
 	weiPerGwei = big.NewInt(1e9)
+	OneGwei    = weiPerGwei
```

Comment:
Can we work around exporting these, since they are mutable?

Reply:
Can we assume the developer using it won't do that?  I find it useful to have these numbers ready to use, rather than figure it out in every package.

Reply:
Can they be deduplicated and commented? I don't understand the naming convention.

Reply:
Helper funcs like this might be a little more in line with the existing api, and are safe from mutation:
```go
// Base converts b base units to wei (*1e18).
func Base(b int) *big.Int {}
// Gwei converts g gwei to wei (*1e9).
func Gwei(g int) *big.Int {}
```
Would that be flexible enough?

Reply:
I like that, want to take a stab it?

Reply:
Sure

Reply:
Pushed